atomic_reactor/build.py

```diff
@@ -166,13 +168,20 @@ def df_path(self):
 
     def set_df_path(self, path):
         self._df_path = path
-        self.set_base_image(df_parser(path).baseimage)
+        dfp = df_parser(path)
+        base = dfp.baseimage
+        if base is None:
+            raise RuntimeError("no base image specified in Dockerfile")
+        self.set_base_image(base)
         logger.debug("base image specified in dockerfile = '%s'", self.base_image)
-        if not self.base_image.tag:
-            self.base_image.tag = 'latest'
+        self.parent_images.clear()
+        for image in dfp.parent_images:
+            self.parent_images[image] = None
 
     def set_base_image(self, base_image):
+        logger.info("setting base image to '%s'", base_image)
```

Comment:
Is this really necessary? We're already pretty noisy.

Reply:
Not really, just was helpful in some of my later debugging of integration tests. To me, verbosity is good as logs are about all we have for debugging builds, and nobody looks at them unless something goes wrong. But I can take this out if you like :)

Reply:
I'm not fond of it but I accept your justification.